src/coreclr/jit/lower.cpp

```diff
@@ -5177,6 +5177,14 @@ bool Lowering::IsFieldListCompatibleWithRegisters(GenTreeFieldList*   fieldList,
                 return false;
             }
 
+            // All SIMDs insertions are not yet supported
```

Comment:
Is this insertion as in `vector = vector.WithElement(index, value)`?

If so, which isn’t supported? The general purpose helper should support all the types, including long

Reply:
It's not supported by the argument creation logic. These cases are very rare (no diffs in this PR).